src/Aspire.Dashboard/Otlp/Storage/TelemetryRepository.cs

```diff
@@ -271,6 +274,11 @@ private void RaiseSubscriptionChanged(List<Subscription> subscriptions)
 
     public void AddLogs(AddContext context, RepeatedField<ResourceLogs> resourceLogs)
     {
+        if (_pauseManager.StructuredLogsPaused)
+        {
+            return;
```

Comment:
Write a trace level message to the logs that structured logs were ignored because of the pause.

Repeat for traces and metrics